src/wrapped_radix_engine_toolkit/high_level/ret_api.rs

```diff
@@ -0,0 +1,420 @@
+        address_of_receiving_account,
+    )
+}
+
+#[uniffi::export]
+pub fn manifest_marking_account_as_dapp_definition_type(
+    account_address: &AccountAddress,
+) -> TransactionManifest {
+    TransactionManifest::marking_account_as_dapp_definition_type(
+        account_address,
+    )
+}
+
+#[uniffi::export]
+pub fn manifest_set_owner_keys_hashes(
+    address_of_account_or_persona: &AddressOfAccountOrPersona,
+    owner_key_hashes: Vec<PublicKeyHash>,
+) -> TransactionManifest {
+    TransactionManifest::set_owner_keys_hashes(
+        address_of_account_or_persona,
+        owner_key_hashes,
+    )
+}
+
+#[uniffi::export]
+pub fn manifest_for_create_fungible_token_with_metadata(
+    address_of_owner: &AccountAddress,
+    initial_supply: Decimal192,
+    metadata: TokenDefinitionMetadata,
+) -> TransactionManifest {
+    TransactionManifest::create_fungible_token_with_metadata(
+        address_of_owner,
+        initial_supply,
+        metadata,
+    )
+}
+
+#[uniffi::export]
+pub fn manifest_for_create_fungible_token(
+    address_of_owner: &AccountAddress,
+) -> TransactionManifest {
+    TransactionManifest::create_fungible_token(address_of_owner)
+}
+
+#[uniffi::export]
+pub fn manifest_for_create_multiple_fungible_tokens(
+    address_of_owner: &AccountAddress,
+) -> TransactionManifest {
+    TransactionManifest::create_multiple_fungible_tokens(address_of_owner)
+}
+
+#[uniffi::export]
+pub fn manifest_for_create_non_fungible_token(
+    address_of_owner: &AccountAddress,
+) -> TransactionManifest {
+    TransactionManifest::create_non_fungible_token(address_of_owner)
+}
+
+#[uniffi::export]
+pub fn manifest_for_create_multiple_non_fungible_tokens(
+    _address_of_owner: &AccountAddress,
+) -> TransactionManifest {
+    todo!()
+}
+
+#[uniffi::export]
+pub fn manifest_stakes_claim(
+    _account_address: &AccountAddress,
+    _stake_claims: Vec<StakeClaim>,
+) -> TransactionManifest {
+    todo!()
+}
+
+#[uniffi::export]
+pub fn manifest_third_party_deposit_update(
+    _to: ThirdPartyDeposits,
+    _owner: &AccountAddress,
+) -> TransactionManifest {
+    todo!()
+}
+
+/// REQUIRES NETWORK CALL (and probable cache)
+/// Requires kotlinx to be setup
+// #[uniffi::export]
+// pub async fn manifest_assets_transfers(
+//     _transfers: AssetsTransfersTransactionPrototype,
+//     _message: Option<Message>,
+// ) -> Result<Manifest> {
+//     todo!()
+// }
+
+#[uniffi::export]
+pub fn updating_manifest_lock_fee(
+    _manifest: TransactionManifest,
+    _address_of_fee_payer: &AccountAddress,
+    _fee: Option<Decimal192>,
+) -> TransactionManifest {
+    todo!()
+}
+
+#[uniffi::export]
+pub fn updating_manifest_add_guarantees(
+    _manifest: TransactionManifest,
+    _guarantees: Vec<TransactionGuarantee>,
+) -> TransactionManifest {
+    todo!()
+}
+
+/// REQUIRES NETWORK CALL (and probable cache)
+/// Requires kotlinx to be setup
+// #[uniffi::export]
+// pub async fn needs_signature_for_depositing(
```

Comment:
I think you will not need that if you take into account this https://github.com/radixdlt/sargon/pull/14/files#r1506278345